node/cli/src/command.rs

```diff
@@ -251,7 +254,17 @@ impl SubstrateCli for RelayChainCli {
 	}
 
 	fn load_spec(&self, id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
-		polkadot_cli::Cli::from_iter([RelayChainCli::executable_name()].iter()).load_spec(id)
+		match id {
```

Comment:
Our version of Polkadot-SDK doesn't have Paseo or Paseo Local, so we have to manually load the json in